conf/messages

```diff
@@ -122,7 +127,47 @@ pfvat.email.change-link.text = Change
 pfvat.amount.title = Amount
 pfvat.amount.change-link.text = Change
 pfvat.address.title = Card billing address
-pfsa.address.change-link.text = Change
+pfvat.address.change-link.text = Change
+
```

Comment:
How come the duplication of same generic messagings? Change, Payment date, Today...

Reply:
I guess in case one needed to be different (it shouldn't so we could argue that it should be condensed down)

Reply:
It was left unrefactored while things sorted themselves out. I think there's some scope for optimising this now but in a different ticket.